tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/SdkBuildTasks/BuildStages/PostBuildTask.cs

```diff
@@ -98,6 +99,15 @@ private void GetApiMapFromProject()
             }
         }
 
+        private void VerifyPropsFile(string apiTag, SdkProjectMetaData sdkProject)
+        {
+            string propsFile = GetApiTagsPropsPath(sdkProject);
+            if(!File.ReadAllText(propsFile).Contains(apiTag))
+            {
+                TaskLogger.LogException(new Exception("Could not find tags "+apiTag+" in the props file "+propsFile));
```

Comment:
@dsgouda we cannot fail the build if we cannot find the apiTag
We cannot afford to block a PR merge because this failed, because apiTag can be updated manually.
So we cannot throw exception here.

Reply:
This function will be called only when an apiTag is non null or empty but is not present in the .props file for some reason, do we still want this to pass?